### PR TITLE
Fixed hard-coded comrade numbers, Disabled buttons that lead to blank screens

### PR DESCRIPTION
**Pull request by HenryDang98**
Comrade numbers are no longer hard-coded
Buttons that lead to blank screens have been disabled
Comrade numbers are now saved and editable after being submitted

### CODE REVIEW

File: PCSA.iml [hunk 1]
```diff
@@ -8,7 +8,7 @@
       </configuration>
     </facet>
   </component>
-  <component name="NewModuleRootManager" inherit-compiler-output="true">
+  <component name="NewModuleRootManager" LANGUAGE_LEVEL="JDK_1_7" inherit-compiler-output="true">
```
Comment (sandarumk) [inline PCSA.iml]: If you use this then when the app is running on Java 1.8 , it will give errors.

File: .idea/.name [hunk 2]
```diff
@@ -0,0 +1 @@
+PCSA
```
Comment (sandarumk) [inline .idea/.name]: This file should not be commited

File: .idea/compiler.xml [hunk 3]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (sandarumk) [inline .idea/compiler.xml]: shouldn't be committed

File: .idea/compiler.xml [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+<project version="4">
```
Comment (sandarumk) [inline .idea/compiler.xml]: shouldn't be committed

**Comment by sandarumk** [comment]
Hi, You have done a good job! But you have committed a large amount of files which shouldn't be committed. Some I mentioned in the comments. If you do a quick google search you will find what types of files which should not be committed. So please make another pull request with these files removed.

File: app/src/main/java/com/peacecorps/pcsa/MainActivityFragment.java [hunk 5]
```diff
@@ -46,7 +48,9 @@ public void onClick(View v) {
         safetyResourceButton.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
-                startActivity(new Intent(getActivity(), SafetyResources.class));
+                Toast.makeText(getActivity(), "This functionality is currently unavailable.", Toast.LENGTH_SHORT).show();
```
Comment (Buddhiprabha) [inline app/src/main/java/com/peacecorps/pcsa/MainActivityFragment.java]: This is a screen containing some static data. Content for this screen is pending from PCSA. Do not remove.

File: app/src/main/java/com/peacecorps/pcsa/reporting/HomeScreen.java [hunk 6]
```diff
@@ -51,16 +52,20 @@ public void onClick(View v) {
         reportingMoreResources.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
-                Intent types = new Intent(HomeScreen.this, Resources.class);
-                startActivity(types);
+                //Commented out because screen is blank.
+                Toast.makeText(HomeScreen.this, "This functionality is currently unavailable.", Toast.LENGTH_SHORT).show();
+                //Intent types = new Intent(HomeScreen.this, Resources.class);
+                //startActivity(types);
```
Comment (Buddhiprabha) [inline app/src/main/java/com/peacecorps/pcsa/reporting/HomeScreen.java]: This is a screen containing some static data. Content for this screen is pending from PCSA. Do not remove.

File: app/src/main/java/com/peacecorps/pcsa/reporting/HomeScreen.java [hunk 7]
```diff
@@ -51,16 +52,20 @@ public void onClick(View v) {
...
 
         reportingConfidentiality.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
-                Intent types = new Intent(HomeScreen.this, Confidentiality.class);
-                startActivity(types);
+                //Commented out because the screen is blank.
+                Toast.makeText(HomeScreen.this, "This functionality is currently unavailable.", Toast.LENGTH_SHORT).show();
+                //Intent types = new Intent(HomeScreen.this, Confidentiality.class);
+                //startActivity(types);
```
Comment (Buddhiprabha) [inline app/src/main/java/com/peacecorps/pcsa/reporting/HomeScreen.java]: This is a screen containing some static data. Content for this screen is pending from PCSA. Do not remove.

**Comment by sandarumk** [comment]
@Buddhiprabha Actually since the activities are not yet implemented, It is ok to disable the functionality and comment the code until PCSA supply the content. It is not stable for an app to have blank screens. He hasn't removed the content. He has commented out and disabled that.
@HenryDang98 Next time when you submit the pull request, mention that when you comment out the activities add a comment that should not removed but only commented out until the functionality is implemented

**Pull request closed without merging**